operator/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,14 @@
+/*
+
```

Comment:
Add regular Kubernetes authors and date

Reply:
this is the output of kubebuilder init, and it is used for autogenerated files only.

I guess that keeping out of the way regular Kubernetes authors and date simplifies management, but eventually we can fix it in follow up PRs

Reply:
Yup, I know what it is used for, thought having it with correct contents would be nice nevertheless, but nothing blocking.